pyperformance/data-files/benchmarks/bm_async_tree/requirements.txt

```diff
@@ -0,0 +1 @@
+pyperf==2.3.1
```

Comment:
``bench_async_func()`` was only added in 2.3.1 for pyperf, but pyperformance is currently using 2.3.0 (according to ``pyperformance/pyperformance/data-files/requirements.txt``). Not sure what the best way to require this version for pyperf is - whether to add it as an ad-hoc requirement like this, or to update the base version used in pyperformance? If the latter, how/who to best do so?